rr_endline_detection/src/main.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright [2018] University of Waterloo Robotics Team
```

Comment:
the file documentation should follow doxygen as noted in our README file

Reply:
done

Reply:
push the code when you've made changes